contentcuration/contentcuration/dev_settings.py

```diff
@@ -21,7 +21,7 @@ def custom_show_toolbar(request):
 else:
     # if debug_panel exists, add it to our INSTALLED_APPS
     INSTALLED_APPS += ('debug_panel', 'debug_toolbar', 'pympler')
-    MIDDLEWARE_CLASSES += ('contentcuration.debug.middleware.CustomDebugPanelMiddleware',)
+    MIDDLEWARE += ('contentcuration.debug.middleware.CustomDebugPanelMiddleware',)
```

Comment:
Was `MIDDLEWARE_CLASSES` deprecated?

Reply:
Yeah, in 1.10. Not sure why using MIDDLEWARE_CLASSES suddenly stopped working though.